thinc/backends/numpy_ops.pyx

```diff
@@ -95,7 +90,7 @@ class NumpyOps(Ops):
         y = self.as_contig(y)
         if out is not None:
             out = self.as_contig(out)
-        return blis.py.gemm(x, y, out=out, trans1=trans1, trans2=trans2, beta=0.)
+        return blis_py.gemm(x, y, out=out, trans1=trans1, trans2=trans2, beta=0.)
```

Comment:
I think we can switch on the dtype of the array here and use `self.cblas().sgemm`/`self.cblas().dgemm`?